conda/recipes/kvikio/meta.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+
+package:
+  name: kvikio
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  noarch: python
+  string: py_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
```

Comment:
Think we need to create a tag (`v22.04.00a` perhaps?) for these variables to be populated

Reply:
I think this has been addressed now